app/templates/components/orders/order-summary.hbs

```diff
@@ -89,11 +89,11 @@
                 </div>
               </td>
               <td>
-                {{#if (eq ticket.type 'free')}}
-                    {{t "Free"}}
-                  {{else}}
-                    {{currency-symbol this.eventCurrency}} {{format-money ticket.price}}
-                  {{/if}}
+                <Orders::TicketPrice
+                  @ticket-type={{ticket.type}}
+                  @display-symbol={{currency-symbol this.eventCurrency}}
+                  @display-price={{format-money ticket.price}}
```

Comment:
Simply pass the ticket. What's the point of creating a component if you'll be doing all calculations here